doc/ref_arch/openstack/chapters/chapter04.rst

```diff
@@ -1773,8 +1628,24 @@ may need to be made for certain Edge clouds depending upon their
 resource capabilities. It should be noted that none of these changes
 affect the definition of OpenStack flavors.
 
-Edge Cloud Deployment
-^^^^^^^^^^^^^^^^^^^^^
+The previous section listed the OpenStack services deployed on the controller nodes depepnding upon
+the control plane distribution. As specified earlier in this chapter, at least 3 controller nodes should be deployed 
+for HA.  Compute nodes may also exist at the sites where controller nodes are deployed.  
+
+Control plane services are not hosted at edge sites. Each edge site can be treated as its own OpenStack AZ. The 
+compute nodes, will host `nova-compute`, a component of the the CCompute Service (Nova), and `neutron-L2-agent`,
+a component of the Network Service (Neutron).
+
+The Edge sites may or may not contain local storage. If the edge sites contain storage, then the Block Storage 
+service (Cinder) is usually deployed to run in an active/active mode with the centrally deployed Block Storage 
+service. Instance images are downloaded and stored locally; they can be downloaded even prior to use.
+
+If the edge site doesn't contain storage then the images would need to be cached from the central site. The instance
+images would be downloaded and cached in the Nova cache on first use; they will then be available for subsequent
```

Comment:
Nova also supports to pre-cache images, but they introduced that in the Ussuri release. Not sure if we want to reflect that here? https://docs.openstack.org/nova/latest/admin/image-caching.html#image-pre-caching